app/helpers/application_helper.rb

```diff
@@ -70,6 +70,13 @@ def format_date(value, time_zone = nil)
     I18n.localize(value, locale: current_user.locale, format: I18n.t('date.input_format.pattern'))
   end
 
+  def snackbar
+    res = content_tag :div, id: "snackbar" do 
+      '' 
+    end
+    res
+  end
```

Comment:
This can be simplified:
```ruby
content_tag :div, "", id: "snackbar"
```

Or just use a plain string since we don't interpolate anything —we declare it HTML safe, so the renderer won't escape entities:

```ruby
%(<div id="snackbar"></div>).html_safe
```

But since there isn't anything special, you can just do it right in HAML:

```haml
  #snackbar
```